Source/src/components/ComponentParticleSystem.cpp

```diff
@@ -469,6 +470,7 @@ void Hachiko::ComponentParticleSystem::Load(const YAML::Node& node)
     shape_section = node[PARTICLE_SECTIONS][SHAPE_SECTION].as<bool>();
     lights_section = node[PARTICLE_SECTIONS][LIGHTS_SECTION].as<bool>();
     renderer_section = node[PARTICLE_SECTIONS][RENDERER_SECTION].as<bool>();
+    //texture_section = node[PARTICLE_SECTIONS][TEXTURE_SECTION].as<bool>();
```

Comment:
I commented this because if we load a scene without texture_section saved previously it will crash. So whenever we have all the scenes saved with the new field "texture_section" then we can uncomment this.

Reply:
Use this:
`texture_section = node[PARTICLE_SECTIONS][TEXTURE_SECTION].IsDefined() ? node[PARTICLE_SECTIONS][TEXTURE_SECTION].as<bool>() : default_value;`

Reply:
Oh Okey, thank you!